Support Adding Tag Expressions To Hook Fns Via Metadata

danielmiladinov opened the pull request:
Resolves #1

danielmiladinov committed:
Support Adding Tag Expressions To Hook Fns Via Metadata

Resolves #1

Signed-off-by: Daniel Miladinov <daniel.miladinov@gmail.com>

Romacoding commented on src/burpless.clj:
```diff
@@ -20,13 +20,21 @@
+  - :after-all - “after all features”
+  - :before - “before each feature”
+  - :after - “after each feature”
+  - :before-step “before each step”
+  - :after-step “after each step”"
```
```suggestion
  - :before-step - “before each step”
  - :after-step - “after each step”"
```

Romacoding reviewed (dismissed):
Great work! Just one small change in a `docstring`.

danielmiladinov committed:
Fix DocString Typos

Each of the other keywords were followed by a dash, so these last two should as well.

Co-authored-by: Roman Ostash <34969764+Romacoding@users.noreply.github.com>

Romacoding reviewed (approved):
:shipit:
